kernelci/rootfs.py

```diff
@@ -50,3 +52,37 @@ def build(name, config, data_path):
     else:
         print("rootfs_type:{} not supported".format(config.rootfs_type))
         return False
+
+
+def _upload_files(api, token, path, input_files):
```

Comment:
There is a very similar function defined in `kernelci/build.py`. We should avoid this kind of a duplication. I think it's a good idea to create a `kernelci/storage.py` module and put the functions responsible for calling the /upload API and import it in both `rootfs.py` and `build.py`.

Reply:
okay added required changes to `rootfs.py` and `build.py`